core/services/ocr2/plugins/ocr2keeper/evm21/streams_lookup.go

```diff
@@ -499,20 +501,46 @@ func (r *EvmRegistry) multiFeedsRequest(ctx context.Context, ch chan<- MercuryDa
 			if resp.StatusCode == http.StatusUnauthorized {
 				retryable = false
 				state = encoding.UpkeepNotAuthorized
-				return fmt.Errorf("at timestamp %s upkeep %s received status code %d from mercury v0.3, most likely this is caused by unauthorized upkeep", sl.Time.String(), sl.upkeepId.String(), resp.StatusCode)
+				return fmt.Errorf("at timestamp %s upkeep %s received status code %d from mercury v0.3, most likely this upkeep does not have access to certain feed IDs", sl.Time.String(), sl.upkeepId.String(), resp.StatusCode)
 			} else if resp.StatusCode == http.StatusBadRequest {
 				retryable = false
 				state = encoding.InvalidMercuryRequest
-				return fmt.Errorf("at timestamp %s upkeep %s received status code %d from mercury v0.3 with message: %s", sl.Time.String(), sl.upkeepId.String(), resp.StatusCode, string(body))
+				return fmt.Errorf("at timestamp %s upkeep %s received status code %d from mercury v0.3, most likely the requested feed ids are not properly hex encoded", sl.Time.String(), sl.upkeepId.String(), resp.StatusCode)
 			} else if resp.StatusCode == http.StatusInternalServerError {
 				retryable = true
 				state = encoding.MercuryFlakyFailure
 				return fmt.Errorf("%d", http.StatusInternalServerError)
+			} else if resp.StatusCode == http.StatusUnprocessableEntity {
```

Comment:
For the time being our team decided we wouldn't allow users to be granted access to feeds that were 'deactivated'  so we may not implement this scenario at this time

Reply:
removed